varnish/tests/common.py

```diff
@@ -0,0 +1,92 @@
+    "varnish.cache_hit",           # metrics where the "MAIN" prefix was removed
+    "varnish.cache_hitpass",       # metrics where the "MAIN" prefix was removed
+    "varnish.cache_miss",          # metrics where the "MAIN" prefix was removed
+    "varnish.backend_conn",        # metrics where the "MAIN" prefix was removed
+    "varnish.backend_unhealthy",   # metrics where the "MAIN" prefix was removed
+    "varnish.backend_busy",        # metrics where the "MAIN" prefix was removed
+    "varnish.fetch_eof",           # metrics where the "MAIN" prefix was removed
+    "varnish.fetch_bad",           # metrics where the "MAIN" prefix was removed
+    "varnish.fetch_none",          # metrics where the "MAIN" prefix was removed
+    "varnish.fetch_1xx",           # metrics where the "MAIN" prefix was removed
+    "varnish.pools",               # metrics where the "MAIN" prefix was removed
+    "varnish.busy_sleep",          # metrics where the "MAIN" prefix was removed
+    "varnish.busy_wakeup",         # metrics where the "MAIN" prefix was removed
+    "varnish.busy_killed",         # metrics where the "MAIN" prefix was removed
+    "varnish.sess_queued",         # metrics where the "MAIN" prefix was removed
+    "varnish.sess_dropped",        # metrics where the "MAIN" prefix was removed
+    "varnish.n_object",            # metrics where the "MAIN" prefix was removed
+    "varnish.n_vampireobject",     # metrics where the "MAIN" prefix was removed
+    "varnish.n_vcl",               # metrics where the "MAIN" prefix was removed
+    "varnish.n_vcl_avail",         # metrics where the "MAIN" prefix was removed
+    "varnish.n_vcl_discard",       # metrics where the "MAIN" prefix was removed
+    "varnish.bans",                # metrics where the "MAIN" prefix was removed
+    "varnish.bans_completed",      # metrics where the "MAIN" prefix was removed
+    "varnish.bans_obj",            # metrics where the "MAIN" prefix was removed
+    "varnish.bans_req",            # metrics where the "MAIN" prefix was removed
+    "varnish.MGT.child_start",
+    "varnish.MGT.child_exit",
+    "varnish.MGT.child_stop",
+    "varnish.MEMPOOL.busyobj.live",
+    "varnish.MEMPOOL.busyobj.pool",
+    "varnish.MEMPOOL.busyobj.allocs",
+    "varnish.MEMPOOL.busyobj.frees",
+    "varnish.SMA.s0.c_req",
+    "varnish.SMA.s0.c_fail",
+    "varnish.SMA.Transient.c_req",
+    "varnish.SMA.Transient.c_fail",
+    "varnish.VBE.boot.default.req",
+    "varnish.LCK.backend.creat",
+    "varnish.LCK.backend_tcp.creat",
+    "varnish.LCK.ban.creat",
+    "varnish.LCK.ban.locks",
+    "varnish.LCK.busyobj.creat",
+    "varnish.LCK.mempool.creat",
+    "varnish.LCK.vbe.creat",
+    "varnish.LCK.vbe.destroy",
+    "varnish.LCK.vcl.creat",
+    "varnish.LCK.vcl.destroy",
+    "varnish.LCK.vcl.locks",
+]
+
+VARNISH_DEFAULT_VERSION = "4.1.7"
+VARNISHADM_PATH = "varnishadm"
+SECRETFILE_PATH = "secretfile"
+DAEMON_ADDRESS = "localhost:6082"
```

Comment:
If this is used to ping docker, we're now using `get_docker_hostname` from `datadog_checks.utils.common`. https://github.com/DataDog/integrations-core/pull/1472